api/src/main/resources/fireCloud.yaml

```diff
@@ -344,3 +462,104 @@ definitions:
       userEmail:
         type: string
         description: User's email
+
+  Workspace:
+    description: ''
+    required:
+      - namespace
+      - name
+      - workspaceId
+      - authorizationDomain
+      - bucketName
+      - createdDate
+      - lastModified
+      - createdBy
+      - attributes
+      - accessLevels
+      - authDomainACLs
+      - isLocked
+    type: object
+    properties:
+      namespace:
+        type: string
+        description: The namespace the workspace belongs to
+      name:
+        type: string
+        description: The name of the workspace
+      authorizationDomain:
+        type: array
+        items:
+          $ref: '#/definitions/ManagedGroupRef'
+        description: The list of groups to form the Authorization Domain (empty if no Authorization Domain is set)
+      workspaceId:
+        type: string
+        description: A UUID associated with the workspace
+      bucketName:
+        type: string
+        description: The name of the bucket associated with the workspace
+      createdDate:
+        type: string
+        format: date-time
+        description: 'The date the workspace was created in yyyy-MM-ddTHH:mm:ss.SSSZZ format'
+      lastModified:
+        type: string
+        format: date-time
+        description: 'The date the workspace was last modified in yyyy-MM-ddTHH:mm:ss.SSSZZ format'
+      createdBy:
+        type: string
+        description: The user who created the workspace
+      attributes:
+        type: object
+        description: Map[String, Attribute]
+      accessLevels:
+        type: object
+        description: Map[String, RawlsGroupRef]
+      authDomainACLs:
+         type: object
+         description: Map[String, RawlsGroupRef]
+      isLocked:
+        type: boolean
+        description: Can the Workspace currently be modified?
+
+  WorkspaceIngest:
```

Comment:
Is this a request object given directly to the API (I think it looks like that)? WorkspaceCreateRequest?

Reply:
This is copied from the FC swagger spec (like everything else in here -- see comment at top). I'd prefer to keep the names consistent.